tools.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+//go:generate go run github.com/vektra/mockery/v2@v2.42.0
+
+//go:generate mockery
```

Comment:
```suggestion
//go:build tools

package main

import (
	_ "github.com/vektra/mockery/v2"
)
```

This moves version management into the regular go.mod, which has better tooling available. I would then create a new file like gen.go and add the following:

```golang
package functions

//go:generate go run github.com/vektra/mockery/v2
```